library/chroma_models.py

```diff
@@ -695,6 +695,7 @@ def forward(
                 input_vec = self.get_input_vec(timesteps, guidance, img.shape[0])
 
                 # kohya-ss: I'm not sure why requires_grad is set to True here
+                # original code: https://github.com/lodestone-rock/flow/blob/c76f63058980d0488826936025889e256a2e0458/src/models/chroma/model.py#L217
```

Comment:
Maybe the other part of the comment might help here, if it is resolved or if we find a way to fix it

https://github.com/lodestone-rock/flow/blob/c76f63058980d0488826936025889e256a2e0458/src/models/chroma/model.py#L195C1-L200C45